OnDemandPassengerTests/VenueServiceTests.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2017 HERE Europe B.V.
+ * All rights reserved.
+ */
+
+import XCTest
+
+class VenueServiceTests: XCTestCase {
```

Comment:
We use Quick framework for our unit tests you can get ViewControllerTests as an example.